examples/resources/env0_team_organization_assignment/resource.tf

```diff
@@ -0,0 +1,15 @@
+resource "env0_team" "team" {
+  name = "team"
+}
+
+resource "env0_custom_role" "custom_role" {
+  name = "custom-role-sample"
+  permissions = [
+    "VIEW_ENVIRONMENT"
+  ]
```

Comment:
Can you please create custom role with organization settings? e.g. view-dashboard? (not sure where the names are taken from in the provider, but I can assist)

Reply:
yeah.. I need to fix the example. Will do that.

Reply:
example fixed.